### PR TITLE
Update model.md: fix typo in URL

### PR DESCRIPTION
**Pull request by evgkrsk**

### CODE REVIEW

File: site/content/userguide/model.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Components are loaded from jsonnet, json, or yaml files. Only jsonnet files prov
 to customize object definitions for different environments. In addition, qbec support loading
 objects from helm charts, and using jsonnet libraries to produce them.
 
-More details can be found in the [authoring components](userguide/authoring/) page.
+More details can be found in the [authoring components](usage/authoring/) page.
```
Comment (harsimranmaan) [inline site/content/userguide/model.md:19]: usage is anchored at the same level as model. Needs a relative reference like `../usage/authoring`

Comment (harsimranmaan) [reply site/content/userguide/model.md:19]: Btw, thanks for raising this

Comment (evgkrsk) [reply site/content/userguide/model.md]: Okay, done.

**Comment by evgkrsk** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by taitken-splunk** [comment]
recheck

**Review by taitken-splunk — APPROVED** [review]

**Comment by taitken-splunk** [comment]
Sorry for the long delay, I am now taking up maintaining this project, so if you could rebase this PR we can get it merged!

**Comment by evgkrsk** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Pull request merged**